main.js

```diff
@@ -11,39 +7,11 @@ addEventListener('load', main)
 async function main() {
   removeEventListener('load', main)
 
-  const theme = setTheme('dark')
-
   if (!document.documentElement.dataset.rendered) {
-    const mermaid = import('./mermaid.js')
```

Comment:
I think mermaid is used in the core spec? It's a graph drawing thing? Does this break it?

Reply:
@glasser right now mermaid is only be used on the actual spec-md sites themselves, and it's still being loaded there.